fuel-core-interfaces/Cargo.toml

```diff
@@ -15,13 +15,7 @@ anyhow = "1.0"
 async-trait = "0.1"
 chrono = { version = "0.4" }
 derive_more = { version = "0.99" }
-fuel-asm = "0.9"
-fuel-crypto = { version = "0.6", default-features = false, features = [ "random" ] }
-fuel-merkle = { version = "0.3" }
-fuel-storage = "0.2"
-fuel-tx = { version = "0.19", default-features = false }
-fuel-types = { version = "0.5", default-features = false }
-fuel-vm = { version = "0.16", default-features = false }
+fuel-vm = { version = "0.16", default-features = false, features = ["random"] }
```

Comment:
hmm this will enable lots of random functions we may not want to allow access to in production code 🤔 

As a tech-debt follow-up, we'll need to figure out a different way to solve the need to use random in fuel-crypto, without enabling it in everything else.